js/libs/form.js

```diff
@@ -1276,7 +1277,44 @@ Fliplet().then(function() {
                   // Update live field
                   field.options = options;
                 },
-                instance: field
+                on: function(eventName, fn) {
+                  var eventListeners = data.fieldEventListeners;
+
+                  if (!eventListeners) {
+                    eventListeners = {};
+                  }
+
+                  eventListeners[field.name] = eventListeners[field.name] || {};
+                  eventListeners[field.name][eventName] = eventListeners[field.name][eventName] || [];
+                  eventListeners[field.name][eventName].push(fn);
+
+                  data.fieldEventListeners = eventListeners;
+                },
+                off: function(eventName, fn) {
+                  var eventListeners = data.fieldEventListeners[field.name][eventName];
+
+                  if (!eventListeners) {
+                    return;
+                  }
+
+                  // No function provided. Clear all hook callbacks.
+                  if (typeof fn === 'undefined') {
+                    eventListeners = [];
+
+                    return;
+                  }
+
+                  // Remove matching handler
+                  eventListeners.forEach(function(handler, i) {
+                    if (handler === fn) {
+                      eventListeners.splice(i, 1);
+                    }
+                  });
+
+                  data.fieldEventListeners[field.name][eventName] = eventListeners;
```

Comment:
Can you confirm if this line is necessary? You ran `eventListeners = [];` above and didn't reassign the empty array back to the event listeners -- because it's an object reference. By the same logic, this line wouldn't be needed either.

Reply:
I forgot to reassign the value in this scope.